server/database/data/label.json

```diff
@@ -0,0 +1,40 @@
+[
+    {
+        
+        "name": "Végétarien",
+        "image": " ../../../client/src/assets/images/Vegetarien.png",

```

Comment:
dynamic images should be stored in public folder.

Reply:
It's done